### ISSUE TITLE
Update to 6.1.0

### ISSUE THREAD

**Issue by Melphios** [YunoHost-Apps/ihatemoney_ynh#64]
Could you please update the package to the last Ihatemoney official version.
It resolve come bug on the currencies side.
Thanks to you

**Comment by Salamandar** [comment]
Hello @Melphios, there’s currently a `testing` branch with 6.1.0 available.

If you’re willing to test it out, you can install it by running this on your server : 

```
sudo yunohost app upgrade ihatemoney -u https://github.com/YunoHost-Apps/ihatemoney_ynh/tree/testing --debug
```

Of course if it seems too risky for you I’ll understand :)

**Comment by Melphios** [comment]
Thanks @Salamandar  the testing version completly solve my cuurencies blocking issue :)

**Comment by Salamandar** [comment]
Excellent ! I tested it at home too, so let's say we merge it to master.

**Comment by zorun** [comment]
FYI we (the upstream project) just released 6.1.1 with more bugfixes for currencies
